doc/examples/boilerplate.py

```diff
@@ -0,0 +1,135 @@
+"""
+A docstring
+
+This can double as a helpful message which explains how the program is run.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+import sys
+from typing import List  # noqa: F401  Flake8 doesn't recognise Python 2 typing.
+
+import libtbx.phil
+from dials.array_family import flex
+import dials.util
+import dials.util.log
+from dials.util.options import OptionParser, flatten_reflections, flatten_experiments
```

Comment:
Do we want to annotate which of these are useful for what tasks?

Reply:
Ooh yes, I was worried you might all be getting sick of my love of comments and docstrings but I'm happy to throw in more.